packages/blade/src/components/Menu/Menu.web.tsx

```diff
@@ -90,21 +93,23 @@ const MenuTree = ({
           isOpen,
         }}
       >
-        {triggerWithReferenceProps}
-        <FloatingList elementsRef={elementsRef} labelsRef={labelsRef}>
-          {isMounted && (
-            <FloatingPortal>
-              <FloatingFocusManager
-                context={context}
-                modal={false}
-                initialFocus={-1}
-                returnFocus={!isNested}
-              >
-                {overlayWithFloatingProps}
-              </FloatingFocusManager>
-            </FloatingPortal>
-          )}
-        </FloatingList>
+        <BaseBox {...metaAttribute({ name: MetaConstants.Menu, testID })}>
```

Comment:
Adding another wrapper may cause issues with integration with other inline components & introduce CSS issues since now in the final html the `trigger` will not be present as how it's rendered in JSX but rather with a Box wrapper. 

We can probably add the Menu meta constants to the Overlay since anyways the trigger itself will not be the "menu" but rather can be any component (eg: button)